### DISCUSSION TITLE
No options to transfer issues to discussions in bulk or through labels

### DISCUSSION THREAD

**Discussion by jb3** [community/community#21564] [Projects and Issues]
<p>Hello,</p>
<p>I’m trying to transfer some issues into discussions on my repository.</p>
<p>According to the support documents located <a href="https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#converting-issues-based-on-labels" rel="noopener nofollow ugc">here</a> and <a href="https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion" rel="noopener nofollow ugc">here</a> there are methods to transfer Issues into Discussions.</p>
<p>Upon trying this on the repositories of my organisation I don’t get this option either for the bulk transfer or single transfer, I’ve verified that discussions are enabled and I can create them.</p>
<p>No button on issue sidebar:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/a/4a16ad03205750732b0d3eaca2e3dc29abc6f5e9.png" data-download-href="/uploads/short-url/azpULnQmqxXDdB7FxUsV2LgNbO9.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/20439493/181065614-0228169b-6024-46bd-baf3-a681d55ac038.png" alt="image" data-base62-sha1="azpULnQmqxXDdB7FxUsV2LgNbO9" width="252" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a16ad03205750732b0d3eaca2e3dc29abc6f5e9_2_252x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a16ad03205750732b0d3eaca2e3dc29abc6f5e9_2_378x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a16ad03205750732b0d3eaca2e3dc29abc6f5e9_2_504x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a16ad03205750732b0d3eaca2e3dc29abc6f5e9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">678×1340 58.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>There is also no button for bulk transfer under the issue labels.</p>
<p>Is this feature not implemented yet? The documentation doesn’t include any mention of the feature not existing so I’m not sure what I need to do to access this feature.</p>
<p>Cheers.</p>

**Comment by kwacky1** [comment]
<p>Is this a private repository? Discussions was made available to private repositories in March.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">

      <a href="https://github.blog/2021-03-09-github-discussions-now-available-for-private-repositories/" target="_blank" rel="noopener" title="09:30PM - 09 March 2021">The GitHub Blog – 9 Mar 21</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/412;"><img src="https://github.blog/wp-content/uploads/2019/03/BlogHeaders_Aligned_PRODUCT_1008x602_x2.png?fit=2016%2C1204" class="thumbnail" width="690" height="412"></div>

<h3><a href="https://github.blog/2021-03-09-github-discussions-now-available-for-private-repositories/" target="_blank" rel="noopener">GitHub Discussions now available for private repositories | The GitHub Blog</a></h3>

  <p>Private repositories can now enable Discussions under “Features” in the repository settings.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jb3** [comment, marked as answer]
<p>Hey <a class="mention" href="/kwacky1">@kwacky1</a>! Thanks for the reply!</p>
<p>This was not a private repository, no. Discussions was available on the repository in question, but bulk transfer options were not present.</p>
<p>I contacted support at the time, and got the answer through there:</p>
<blockquote>
<p>There was a change recently that turned off this feature. There is a bug in the system that can cause comments to disappear occasionally when issues are converted to discussions . This change was intentional to allow the team time to diagnose and resolve that issue so no comments are lost in the conversion.</p>
</blockquote>
<blockquote>
<p>The engineering team is still researching what is going on with this and getting ready to implement more logging to collect data. We do not have an ETA on when this will be resolved, but it is definitely in the right hands.</p>
</blockquote>
<p>Thanks again for the reply!</p>
<p>Joe</p>

**Comment by thorwhalen** [comment]
Has this issue been solved? I can't find a way to do bulk issue->discussion conversions still, except through github's API.
